metrics/repl.lag/lag.go

```diff
@@ -115,70 +130,122 @@ func (c *Lag) Help() blip.CollectorHelp {
 	}
 }
 
+// Prepare prepares lag collectors for all levels in the plan that contain the "repl.lag" domain
 func (c *Lag) Prepare(ctx context.Context, plan blip.Plan) (func(), error) {
 LEVEL:
-	for _, level := range plan.Levels {
+	for levelName, level := range plan.Levels {
 		dom, ok := level.Collect[DOMAIN]
 		if !ok {
 			continue LEVEL // not collected in this level
 		}
+		cleanup, err := c.prepareLevel(levelName, plan.MonitorId, plan.Name, dom.Options)
```

Comment:
Inline `prepareLevel` here because its logic _is_ `Prepare`. Just blogged about this: https://hackmysql.com/post/fallacy-of-small-functions/.  Factoring out `prepareBlipHeartbeatLag` is probably ok since it actually has some "depth", but `preparePFS` can probably be inlined too: it's 2 lines of code, not counting the `return`.

Reply:
Will refactor. I have read the blog post earlier, nice read!